src/GeoStatsBase.jl

```diff
@@ -291,7 +292,7 @@ export
   mean, var, quantile,
 
   # plot recipes
-  cornerplot,
+  cornerplot, histograms,
```

Comment:
I think we don't have a `histograms` name to export, right? We are just making the EmpiricalHistogram a first-class citizen to be able to define plot recipes for example. Can you erase this export?

Reply:
I just tried exporting it because I couldn't get the plot thing to work. Now I know why. I'll erase this export.